### ISSUE TITLE
Glide#with#pauseRequests will pause all load request?

### ISSUE THREAD

**Issue by codepackage** [bumptech/glide#848]
https://github.com/bumptech/glide/issues/582
Find and load cached Image in Glide #509
https://github.com/bumptech/glide/issues/90

I search for the issues.but I don't think I agree with the result.So I submit a new issue.~_~

if the image already in the memory cache, it cannot load when listview scrolling. 
I use "pauseRequests", It will pause all requests. so, I just saw "placeholder" picture when listview scrolling. But I just want it pause the request in "EngineRunnable", don't want it pause the request from the memory cache.

**Comment by TWiStErRob** [comment]
Hmm, that seems reasonable, let's see what @sjudd (the author) has to say about this.

In the meantime I think you can simulate this behavior via:
```java
.diskCacheStrategy(NONE) // force going to the network
.using(new NetworkDisablingLoader()) // forbid network access
```
See #639 where these lines go, it's very similar to that, just ignore the `thumbnail` part and the 3g request.

The result should be that memory cached loads will complete, but memory cache misses will try disk cache (which is disabled) and then try network and fail. This will result in `listener.onException` being called and `.error()` drawable being displayed (if not null), so be careful if you have logic there.

**Comment by codepackage** [comment]
I load all pictures from local. Do not load them from network. I focus on the User Experience when scrolling fast。If glide can load picture from memory cache, It can show the pictures in memory cache even when scrolling fast.

**Comment by TWiStErRob** [comment]
Sorry, replace the word "network" with "source" (network, file, APK/res ...) above everywhere, Glide is very generic in that sense. The point is that it won't allow loading/decoding if it's not already loaded.

**Comment by sjudd** [comment]
I agree this is a worthy enhancement, it's not totally straightforward because pausing/restarting requests is handled at a much higher level.

**Comment by zhengluqiang** [comment]
@TWiStErRob I still want to know,will the new version solve this @ #582 , if use `pauseRequests` when listview scrolling,the images already in the memory cache can load instead of  `placeholder`? Or just don't need to think about this cause listview can scroll smoothly When using Glide by default usage?

**Comment by TWiStErRob** [comment]
@zhengluqiang I don't think anything has changed regarding this, otherwise this issue would likely be resolved. I don't see a point of pausing requests unless you have a performance issue. You can also try to [tie pause/resume to the speed at which the fling is happening](https://github.com/TWiStErRob/glide-support/blob/master/src/glide3/java/com/bumptech/glide/supportapp/github/_864_staggered_grid/PauseOnFling.java). While it is fast the user likely wouldn't see the image, but as it shows down each item will be on-screen longer and longer.
